tgui/packages/tgui/interfaces/ShuttleManipulator.jsx

```diff
@@ -144,80 +116,87 @@ const ShuttleManipulatorStatus = (props) => {
   );
 };
 
-const ShuttleManipulatorTemplates = (props) => {
-  const { act, data } = useBackend<ShutteManipulatorData>();
-  const templateObject = data.template_data;
-  const selected = data.selected;
-  const [selectedTemplateId, setSelectedTemplateId] = useLocalState(
-    'templateId',
-    templateObject[0]?.shuttle_id ?? 0
-  );
-  const actualTemplate = templateObject.find(
-    (x) => x.shuttle_id === selectedTemplateId
+export const ShuttleManipulatorTemplates = (props) => {
+  const { act, data } = useBackend();
+  const templateObject = data.templates || {};
+  const selected = data.selected || {};
+  const [selectedTemplateId, setSelectedTemplateId] = useState(
+    Object.keys(templateObject)[0],
   );
+  const actualTemplates = templateObject[selectedTemplateId]?.templates || [];
+
   return (
     <Section>
       <Flex>
         <Flex.Item>
           <Tabs vertical>
-            {templateObject.map((template) => (
+            {map(templateObject, (template, templateId) => (
```

Comment:
This isn't neccesary

Reply:
Reversion results in 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/13d44efb-f1f8-4d6a-ad0a-ea4f7e2666ce)
and ultimately its just what upstream has here: https://github.com/tgstation/tgstation/blob/master/tgui/packages/tgui/interfaces/ShuttleManipulator.jsx#L107